xee/ext.py

```diff
@@ -482,7 +496,15 @@ def image_to_array(
         **kwargs,
     }
     raw = common.robust_getitem(
-        pixels_getter, params, catch=ee.ee_exception.EEException
+        pixels_getter,
+        params,
+        catch=ee.ee_exception.EEException,
+        max_retries=self.tile_fetch_kwargs.get(
```

Comment:
You can replace `max_retries=` and `initial_delay=` with `**self.tile_fetch_kwargs`.